### ISSUE TITLE
No Mention of isAuthenticated() in docs

### ISSUE THREAD

**Issue by prakhar897** [jaredhanson/passport#683]
Hi,
The method isAuthenticated() is not mentioned anywhere in the docs.Also is there any method named isUnauthenticated() too?

**Comment by em92** [comment]
> Also is there any method named isUnauthenticated() too?

Yes.
https://github.com/jaredhanson/passport/blob/597e289d6fa27a2c35d16dd411de284123e3817e/lib/http/request.js#L92-L100

**Comment by bfischer1121** [comment]
Yes this is a big omission since it's kind of the point of the library :)

**Comment by erick2014** [comment]
So what's the bug here?

**Comment by brenc** [comment]
Just started using Passport and really struggled with this. Some mention in the docs would be good!

**Comment by uzair004** [comment]
Can't believe this issue is not fixed yet, it's been 2.5 years but they haven't updated the docs to solve the problem.

also the examples page for passport-local doesn't exist anymore.

**Comment by christian-hawk** [comment]
@uzair004 it's open-source, so I understand your PR is welcome.

**Comment by uzair004** [comment]
> @uzair004 it's open-source, so I understand your PR is welcome
There is already a PR to solve the problem but it's not merged yet, it's been more then 2 years
